src/test/java/org/llorllale/cactoos/matchers/MatcherOfTest.java

```diff
@@ -44,7 +47,11 @@
     public void matchesFunc() {
         new Assertion<>(
             "matches when arg satisfies the predicate",
-            new MatcherOf<>(x -> x > 5),
+            new MatcherOf<>(x -> x > 5,
+                    desc -> desc.appendText(
+                        new UncheckedText("matches when arg satisfies the predicate").asString()
```

Comment:
@ruca905 wrapping this in an `UncheckedText` is not needed, just use `appendText` with the string directly. Also don't bother about putting something complex here, just write `">5"` for example, this is a test and should stay simple since this part is not tested here